ElementX/Sources/Screens/HomeScreen/View/HomeScreenUserMenuButton.swift

```diff
@@ -31,7 +32,11 @@ struct HomeScreenUserMenuButton: View {
                     Label {
                         Text(L10n.commonSettings)
                     } icon: {
-                        settingsIconImage
+                        if context.viewState.showSettingsMenuOptionBadge {
+                            Image(asset: Asset.Images.settingsIconWithBadge)
```

Comment:
So you can use this here, although I'm not sure it really gives you much in this particular instance, only for consistency.
```suggestion
                            CompoundIcon(customImage: Asset.Images.settingsIconWithBadge.swiftUIImage)
```